battleship.py

```diff
@@ -163,17 +162,25 @@ def drawGrid(data, canvas, grid, showShips):
 Returns: bool
 '''
 def isVertical(ship):
-    return
-
+    if ship[0][1]==ship[1][1]==ship[2][1]:
+        ship.sort()
+        for row in ship:
+            if ship[0][0]+1==ship[1][0]==ship[2][0]-1:
+                return True
+    return False
 
 '''
 isHorizontal(ship)
 Parameters: 2D list of ints
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
-
+    if ship[0][0]==ship[1][0]==ship[2][0]:
+        ship.sort()
+        for col in ship:
+            if ship[0][1]+1==ship[1][1]==ship[2][1]-1:
+                return True
+    return False

```

Comment:
@priyusha99 I have no problem with the logic you have to build. Try to re-think and make the code in fewer lines.